go.mod

```diff
@@ -1,3 +1,16 @@
-module github.com/antsanchez/gozerobounce
+module github.com/zerobounce-llc/zerobouncego
 
-go 1.15
\ No newline at end of file
+go 1.20
```

Comment:
does this mean older versions of go are not supported? afaik this has just been released, right?